testing/citest/spinnaker_testing/scrape_spring_config.py

```diff
@@ -81,7 +81,7 @@ def infer(json):
   return expr_dict
 
 
-def scrape_spring_config(url, timeout=60, empty_if_404=True):
+def scrape_spring_config(url, timeout=60, empty_if_404=True, bearer_auth_token=None):
```

Comment:
Seems like bearer_auth_token=None should instead be a list of headers.
The caller with a bearer auth token would pass in a list containing the authorization header.
If that's hard it could be a refactorization for later if other headers are needed.
But if it is all the same, it feels like this should be generalized.